usr/share/drlm/conf/default.conf

```diff
@@ -675,7 +722,7 @@ REAR_SERVICES_SUSE15="rpcbind nfs"
 ############### ----- SUDO COMMANDS
 #
 ########
-SUDO_CMDS_DRLM=( 'mount' 'umount' 'rm' 'mkdir' 'vgs' 'stat' 'cat /etc/rear/drlm.token' )
+SUDO_CMDS_DRLM=( 'mount' 'umount' 'rm' 'mkdir' 'vgs' 'stat' 'cat /etc/rear/drlm.token' 'git' 'make -C /var/lib/drlm/*' )
```

Comment:
The wildcard `*` permits run make in any directory. You can perform `make -C /var/lib/drlm/../../../etc/` and will match the sudo rule.